yarn.lock

```diff
@@ -1,16191 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
yarn.lock file should exists in the repo and is not meant to be edited manually, please don't delete this file

Reply:
Regenerated the yarn.lock and merged it check [3eabd74](https://github.com/Platzi-Master-C9/booking-ui/pull/190/commits/3eabd740f6403543c9843a00a8492f0023891a98)